app/views/admin/instances/index.html.haml

```diff
@@ -15,10 +15,14 @@
   .filter-subset
     %strong= t('admin.instances.moderation.title')
     %ul
-      %li= filter_link_to t('admin.instances.moderation.all'), limited: nil
-
-      - unless limited_federation_mode?
-        %li= filter_link_to t('admin.instances.moderation.limited'), limited: '1'
+      - if limited_federation_mode?
+        %li= filter_link_to t('admin.instances.moderation.allowed'), status: nil
+      - else
+        %li= filter_link_to t('admin.instances.moderation.all'), status: nil
+        %li= filter_link_to t('admin.instances.moderation.not_limited'), status: 'not_limited'
+        %li= filter_link_to t('admin.instances.moderation.noop'), status: 'noop'
```

Comment:
The localisation here for "noop" is "Filtered", which seems like the most reasonable value, because showing a moderator "NOOP" in the menu just doesn't really make sense, and usually this severity is used in combination with the `reject_*` fields in domain blocks, which are effectively "filters"